pyaedt/emit_core/results/revision.py

```diff
@@ -732,3 +732,23 @@ def set_emi_category_filter_enabled(self, category: EmiCategoryFilter, enabled:
             raise RuntimeError("This function is only supported in AEDT version 2024 R1 and later.")
         engine = self.emit_project._emit_api.get_engine()
         engine.set_emi_category_filter_enabled(category, enabled)
+
+    @property
+    def license_session(self):
+        """Get a license_session object.
+
+        A license session can be started with checkout(), and ended with checkin().
+        The `with` keyword can also be used, where checkout() is called on enter, and checkin() is called on exit.
```

Comment:
```suggestion
        You can start a license session with the ``checkout()`` method and end it with the ``checkin()`` method.
        You can also use the ``with`` keyword, where the ``checkout()`` method is called on enter and the ``checkin()``
        method is called on exit.
```

Reply:
@bryankaylor The previous converstion that you marked as resolved has suggestions to bring the content inline with our style guide. Please incorporate the suggested changes.